### PR TITLE
Add RetentionContest -> CandidateId validation

### PR DESCRIPTION
**Pull request by cap10morgan**
Validates that all RetentionContest elements have CandidateId child elements.

[Pivotal card](https://www.pivotaltracker.com/story/show/113421645)

### CODE REVIEW

File: src/vip/data_processor/validation/v5/retention_contest.clj [hunk 1]
```diff
@@ -0,0 +1,16 @@
+(ns vip.data-processor.validation.v5.retention-contest
+  (:require [korma.core :as korma]
+            [vip.data-processor.db.postgres :as postgres]
+            [vip.data-processor.validation.v5.util :as util]))
```
Comment (tie-rack) [inline src/vip/data_processor/validation/v5/retention_contest.clj]: You don't need to require korma or postgres here.

**Comment by tie-rack** [comment]
And add the validation to the list in v5.clj.

File: src/vip/data_processor/validation/v5/retention_contest.clj (hunk 1 above)
Comment (cap10morgan) [inline src/vip/data_processor/validation/v5/retention_contest.clj]: Ah, that's now true! (I started this before you created `build-xml-tree-value-query-validator`.)

**Comment by tie-rack** [comment]
LGTM

**Pull request merged**